tm2/pkg/sdk/auth/params.go

```diff
@@ -67,6 +72,7 @@ func DefaultParams() Params {
 		DefaultSigVerifyCostSecp256k1,
 		DefaultGasPricesChangeCompressor,
 		DefaultTargetGasRatio,
+		crypto.AddressFromPreimage([]byte(DefaultFeeCollectorName)),
```

Comment:
`gno.DerivePkgAddr("gno.land/r/sys/txfees")`

Reply:
I cannot import `gnovm` pkgs from `tm2`. Perhaps `DerivePkgAddr` is not in the correct location? Additionally, having a reference to a `gno.land` package as a default value inside `tm2` seems strange.

Reply:
no, please, keep exactly your line, it's the good odefault value we want for this generic tm2 module.

Reply:
You guys are right 🤦‍♂️ 
This is the SDK code -- shouldn't have any traces of gno in it :)

Ignore my comment